### ISSUE TITLE
Project not working. Video doesnt load.

### ISSUE THREAD

**Issue by animemandir** [SilentTush/Animy#1]
I think the api might be changed, from where do you fetch that storage.googleapis.com video links. 
many sites have stopped using those- 9anime and animesuge have also stopped using those.
you should scrape data from other sites. check piracy.moe for anime sites.

**Comment by SilentTush** [comment]
yes, i know there's some issue in fetching video source coz the the site i was using for scrapping (kimanime) changed their servers, so will fix it soon, will probably have to switch to some other website. I'll try the sites mentioned.
